dbus/org.nemo.transferengine.xml

```diff
@@ -48,6 +48,21 @@
             <arg direction="out" type="i" name="transferId"/>
         </method>
 
+        # create transient Download entry
+        <method name="createTransientDownload">
+            <arg direction="in" type="s" name="displayName"/>
+            <arg direction="in" type="s" name="applicationIcon"/>
+            <arg direction="in" type="s" name="serviceIcon"/>
+            <arg direction="in" type="s" name="filePath"/>
+            <arg direction="in" type="s" name="mimeType"/>
+            <arg direction="in" type="x" name="expectedFileSize"/>
+            <arg direction="in" type="b" name="transient"/>
```

Comment:
If the method is "createTransient..." it should already imply that transient is true. So either the code should be adjusted to call either createDownload or this method, or rename this method not to imply transient.